src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -158,9 +158,7 @@ class DocBaseViewer extends BaseViewer {
         super.setup();
 
         this.docEl = this.createViewer(document.createElement('div'));
-        this.docEl.setAttribute('aria-label', __('document_label'));
         this.docEl.classList.add('bp-doc');
-        this.docEl.tabIndex = '0';
```

Comment:
This logic has been here since 2019. Its removal will likely break keyboard scrolling and possibly other desired behaviors.